project_task_git_branch/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Task GIT Branch
+===============
+
+This module allows you to manage your git .
```

Comment:
replace:
This module allows you to manage your git .
with
This module allows you to manage your git within the tasks of a project.